lib/vsc/administration/slurm/sync.py

```diff
@@ -653,7 +652,13 @@ def slurm_vo_accounts(account_page_vos, slurm_account_info, clusters, host_insti
     return commands
 
 
-def slurm_project_users_accounts(project_members, active_accounts, slurm_user_info, clusters):
+def slurm_project_users_accounts(
+    project_members,
+    active_accounts,
```

Comment:
this is ugly. black I assume?

Reply:
Otherwise we exceed the line length and I think splitting arguments in two lines is even worse than this.

Reply:
I will die on this hill, btw.